definitions/marts/bigquery_marts/npq/npq_enrolments.sqlx

```diff
@@ -9,7 +9,79 @@ config {
+        employment_role: "Shows the employment role of non-school users. Text field.",
+        employment_type: "For non-school users, shows the type of institution they are employed in. Possible fields: Local authority supply teachers, Local authority virtual school, Young offender institute, Hospital school, Other.",
+        declaration_user_id: "",
+        profile_user_id: "",
+        training_status: "Pulled from participant profile which is created when application is accepted by lead provider.",
+        schedule_identifier: "This indicates which sub-cohort or tranche the participant commenced training within an annual cohort. For NPQ, the schedule identifier also distinguishes between the leadership and specialist NPQ types. Pulled from participant profile which is created when application is accepted by lead provider.",
+        profile_updated_at: "Refers to the latest updated date for the participant profile.",
+        started_declaration_id: "ID of the participant's started declaration.",
+        started_declaration_state: "State of the participant's started declaration.",
+        started_declaration_date: "Date evidence was submitted for participant's started declaration.",
+        funded_start_declaration: "True if the declaration is in an eligible, payable, or paid state.",
+        retained1_declaration_id: "ID of the participant's retained-1 declaration.",
+        retained1_declaration_state: "State of the participant's retained-1 declaration.",
+        retained1_declaration_date: "Date evidence was submitted for participant's retained-1 declaration.",
+        funded_retained1_declaration: "True if the declaration is in an eligible, payable, or paid state.",
+        retained2_declaration_id: "ID of the participant's retained-2 declaration.",
+        retained2_declaration_state: "State of the participant's retained-2 declaration.",
+        retained2_declaration_date: "Date evidence was submitted for participant's retained-2 declaration.",
+        funded_retained2_declaration: "True if the declaration is in an eligible, payable, or paid state.",
+        completed_declaration_id: "ID of the participant's completed declaration.",
+        completed_declaration_state: "State of the participant's completed declaration.",
+        funded_completed_declaration: "True if the declaration is in an eligible, payable, or paid state.",
+        completed_declaration_date: "Date evidence was submitted for participant's completed declaration.",
+        latest_outcome_state: "State of participant's latest outcome for their NPQ. Possible fields: Passed, Failed, Voided.",
+        latest_outcome_date: "Date of completion for the participant's latest outcome.",
+        count_of_fails: "Counts the number of Failed outcome states for a participant.",
+        cohort_id: "ID of cohort.",
+        npq_lead_provider_id: "ID of Lead Provider.",
+        participant_identity_id: "",

```

Comment:
Might be worth saying what a participant_identity_id is and how it differs from profile id